### PR TITLE
Fix #3212: HTML Builders crashes with docutils-0.13

### LINKED ISSUE #3212: [docutils 0.13.1] assert not self.context
Following traceback is found when building buildbot documentation after upgrade to docutils 0.13.1

```
Traceback (most recent call last):
  File "/Users/ptardy/dev/bb/buildbot/sandbox/lib/python2.7/site-packages/sphinx/cmdline.py", line 296, in main
    app.build(opts.force_all, filenames)
  File "/Users/ptardy/dev/bb/buildbot/sandbox/lib/python2.7/site-packages/sphinx/application.py", line 333, in build
    self.builder.build_update()
  File "/Users/ptardy/dev/bb/buildbot/sandbox/lib/python2.7/site-packages/sphinx/builders/__init__.py", line 251, in build_update
    'out of date' % len(to_build))
  File "/Users/ptardy/dev/bb/buildbot/sandbox/lib/python2.7/site-packages/sphinx/builders/__init__.py", line 322, in build
    self.write(docnames, list(updated_docnames), method)
  File "/Users/ptardy/dev/bb/buildbot/sandbox/lib/python2.7/site-packages/sphinx/builders/__init__.py", line 360, in write
    self._write_serial(sorted(docnames), warnings)
  File "/Users/ptardy/dev/bb/buildbot/sandbox/lib/python2.7/site-packages/sphinx/builders/__init__.py", line 368, in _write_serial
    self.write_doc(docname, doctree)
  File "/Users/ptardy/dev/bb/buildbot/sandbox/lib/python2.7/site-packages/sphinx/builders/html.py", line 452, in write_doc
    self.docwriter.write(doctree, destination)
  File "/Users/ptardy/dev/bb/buildbot/sandbox/lib/python2.7/site-packages/docutils/writers/__init__.py", line 80, in write
    self.translate()
  File "/Users/ptardy/dev/bb/buildbot/sandbox/lib/python2.7/site-packages/sphinx/writers/html.py", line 48, in translate
    self.document.walkabout(visitor)
  File "/Users/ptardy/dev/bb/buildbot/sandbox/lib/python2.7/site-packages/docutils/nodes.py", line 187, in walkabout
    visitor.dispatch_departure(self)
  File "/Users/ptardy/dev/bb/buildbot/sandbox/lib/python2.7/site-packages/docutils/nodes.py", line 1895, in dispatch_departure
    return method(node)
  File "/Users/ptardy/dev/bb/buildbot/sandbox/lib/python2.7/site-packages/docutils/writers/_html_base.py", line 671, in depart_document
    assert not self.context, 'len(context) = %s' % len(self.context)
AssertionError: len(context) = 4
```

You can reproduce by downloading a copy of buildbot 
```
git clone git@github.com:buildbot/buildbot.git
cd buildbot
pip install -e ./master[docs]'
make docs
```

### PR DESCRIPTION
**Pull request by tk0miya**
refs #3212.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,8 +15,8 @@ env:
     - PYTHONFAULTHANDLER=x
     - PYTHONWARNINGS=all
   matrix:
-    - DOCUTILS=0.11
```
Comment (tk0miya) [inline .travis.yml:18]: At this PR, I removed docutils-0.11 from test matrix to prevent to increase test time.
@shimizukawa what do you think?

Comment (shimizukawa) [reply .travis.yml:18]: It's OK to me.
IMO, it is sufficient to keep only the last two versions.

Comment (tk0miya) [reply .travis.yml:18]: Thanks!

File: sphinx/writers/html.py [hunk 2]
```diff
@@ -532,6 +533,16 @@ def visit_image(self, node):
                         node['height'] = str(size[1])
         BaseTranslator.visit_image(self, node)
 
+    # overwritten
+    def depart_image(self, node):
+        if docutils.__version__ >= "0.13":
```
Comment (mgeier) [inline sphinx/writers/html.py:538]: This comparison doesn't always work:

```python
>>> '0.9' >= '0.13'
True
```

Comment (tk0miya) [reply sphinx/writers/html.py:538]: Good point. I post a new PR for this: #3225.

**Pull request merged**